web/docs/setup-vast/monitor.md

```diff
@@ -140,6 +140,7 @@ The following list describes all available metrics keys:
 |`active-store.lookup.hits`|The number of results of a query in an active store.|#events|🔎🪪💾|
 |`passive-store.lookup.runtime`|The number of results of a query in a passive store.|#events|🔎🪪💾|
 |`passive-store.lookup.hits`|The number of results of a query in a passive store.|#events|🔎🪪💾|
+|`passive-store.init.runtime`|Time until the store is ready serve queries.|nanoseconds|🔎🪪💾|
```

Comment:
```suggestion
|`passive-store.init.runtime`|Time until the store is ready serve queries.|nanoseconds|🪪💾|
```

A query UUID makes no sense in the store load runtime metrics metadata.

Reply:
fixed, missed these pretty pictures. "issuer" is also not a thing so I removed that as well.